<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTabPageForm</class>
 <widget class="QWidget" name="MainTabPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="10,5,5,20" columnminimumwidth="0,0,50,0">
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="SharedStashItemCountLabel">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="InfiniteStashTitleLabel">
     <property name="text">
      <string>Infinite Stash Items</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="SharedStashItemCountTitleLabel">
     <property name="text">
      <string>Total Items in Stash:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="SharedStashListWidget" name="Torchlight2SharedStashListWidget"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="SharedStashTitleLabel">
     <property name="text">
      <string>Torchlight 2 Shared Stash Items</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="InfiniteStashTreeView" name="InfiniteStashItemsTreeView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SharedStashListWidget</class>
   <extends>QListWidget</extends>
   <header>sharedstashlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>InfiniteStashTreeView</class>
   <extends>QTreeView</extends>
   <header>infinitestashtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
